proxy/Cargo.toml

```diff
@@ -70,6 +70,7 @@ postgres-native-tls.workspace = true
 
 workspace_hack.workspace = true
 tokio-util.workspace = true
+lazy_static = "1.4.0"
```

Comment:
we define deps on workspace level (look at all lines in the file)

Reply:
I'm sorry, cargo added it for me and I haven't checked